pom.xml

```diff
@@ -255,6 +274,11 @@
                 <version>2.9</version>
             </plugin>
             <plugin>
+                <groupId>org.jacoco</groupId>
```

Comment:
I don't see the report in the Maven site after running `mvn clean site:site`. Am I missing something?